Makefile

```diff
@@ -1,9 +1,11 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= telemetry-operator:latest
```

Comment:
let's rename this also to telemetry-manager

Reply:
image name renamed to telemetry manager